sdl2_mixer/VITABUILD

```diff
@@ -1,33 +1,40 @@
 pkgname=sdl2_mixer
 pkgver=2.6.3
-pkgrel=1
+pkgrel=2
+gitrev=eb005518b0085244630ccfb41977f818af45d2ea
 url="https://github.com/libsdl-org/SDL_mixer"
-source=("https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-${pkgver}.tar.gz"
-        "pkg-config-fix.patch")
+source=("https://github.com/libsdl-org/SDL_mixer/archive/${gitrev}.tar.gz"
```

Comment:
```suggestion
source=("git+https://github.com/libsdl-org/SDL_mixer.git#commit=${gitrev}"
```
This would make it use git instead of downloading as an archive. This will require the cd commands to be updated.